src/tools/performance_monitoring_tools.py

```
@@ -0,0 +1,349 @@
+                    "severity": alert.severity,
+                    "message": alert.message,
+                    "timestamp": alert.timestamp.isoformat(),
+                    "recommendations": alert.recommendations,
+                }
+                for alert in recent_alerts[-10:]  # Last 10 alerts
+            ]
+
+        return json.dumps(response, indent=2)
+
+    except Exception as e:
+        logger.error(f"Error in get_performance_summary: {str(e)}", exc_info=True)
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)